src/main/resources/db/migration/V2__FixPasswordsToBeHashedSecure.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE USERS ALTER COLUMN PASSWORD RENAME TO HASH;
```

Comment:
Migration here. This is a trivial change, initially I thought we would have to create another table to store token expiry, but as I understand the stateless JWT authenticator can do it cryptographically without a store for the tokens.